server/src/main/java/org/elasticsearch/common/unit/RelativeByteSizeValue.java

```diff
@@ -103,4 +108,27 @@ public String getStringRep() {
             return absolute.getStringRep();
         }
     }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeBoolean(isAbsolute());
+        if (isAbsolute()) {
+            assert absolute != null;
+            absolute.writeTo(out);
+        } else {
+            assert ratio != null;
+            out.writeDouble(ratio.getAsPercent());
```

Comment:
Could we make `RatioValue implements Writeable` too rather than relying on knowing that `getAsPercent` is a faithful representation of the value on the wire?